### ISSUE TITLE
Support installation via Homebrew

### ISSUE THREAD

**Issue by cthoyt** [ontodev/robot#910]
It would be great if it were possible to install robot with homebrew (https://brew.sh/) with a command like `brew install robot`. This would greatly simplify the need to putz around with the fact that ROBOT is implemented in java, picking a place for the JAR, updating your classpath/system path, etc. that are described in this section of the [getting started page](https://robot.obolibrary.org/):

<img width="646" alt="Screen Shot 2021-09-16 at 11 32 37" src="https://user-images.githubusercontent.com/5069736/133588244-ee1337ae-f5e9-4e19-9d58-520698a85f9d.png">

**Comment by jamesaoverton** [comment]
See also this older issue about packaging #827.

**Comment by cthoyt** [comment]
@jamesaoverton thanks for the link, I searched the issues to see if anyone had mentioned brew/homebrew before and didn't see this

**Comment by jamesaoverton** [comment]
@cthoyt You're right. I wanted to link the previous issue, but it does not mention homebrew, so this is not a duplicate.

Do you (or anyone reading) have experience packaging for homebrew?

**Comment by cthoyt** [comment]
Not really. I tried doing it once and got a bit discouraged.

**Comment by balhoff** [comment]
The formula for Jena (another Java-based tool) might be instructive: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/jena.rb
